flamedisx/utils.py

```diff
@@ -203,3 +206,17 @@ def values_to_constants(kwargs):
 def wilks_crit(confidence_level):
     """Return critical value from Wilks' theorem for upper limits"""
     return stats.norm.ppf(confidence_level) ** 2
+
+
+@export
+def run_command(command):
+    """Run command and show its output in STDOUT"""
+    # Is there no easier way??
```

Comment:
For cleaner syntax and simple operations, it is possible to use `os.system(command)` instead as suggested in https://stackoverflow.com/questions/4813238/difference-between-subprocess-popen-and-os-system. However, official documentation for `os` module suggests using `subprocess` module https://docs.python.org/3/library/os.html. So, keeping software longevity in mind, it is probably better to use `subprocess`.

Reply:
Thanks Pueh; I tried os.system, but it didn't show me the output of the command while it was running. Since the cloning of BBF takes some time, I thought it was nicer to show the output as it happened, like ! from ipython. But this indeed comes at the cost of some extra complication.